fluent-dedent/src/index.js

```diff
@@ -0,0 +1,36 @@
+// A blank line may contain spaces and tabs.
+const RE_BLANK = /^[ \t]*$/;
+
+/**
+ * Template literal tag for dedenting Fluent code.
+ *
+ * Strip the indent of the last line from each line of the input. Remove the
+ * first and the last line from the output. The snippet must start on a new
+ * line and it must end on a line of its own, with the closing delimiter on a
+ * next line.
+ *
+ * @param {Array<string>} strings
+ * @returns string
+ */
+export default function ftl(strings) {
+  let lines = strings[0].split("\n");
```

Comment:
Should this also test for extra arguments?

Reply:
Yes, it looks like I forgot to push the second commit which adds this :) I've updated the PR just now.